fuzzers/cargo_fuzz/Makefile.toml

```diff
@@ -0,0 +1,37 @@
+[env]
+
+[tasks.unsupported]
+script_runner="@shell"
+script='''
+echo "Cargo-make not integrated yet on this"
+'''
+
+[tasks.install_llvm_tools]
+command = "rustup"
+args = ["toolchain", "install", "nightly", "--component", "llvm-tools-preview"]
+
+
+[tasks.install_cargo_fuzz]
+command = "cargo"
+args = ["install", "cargo-fuzz"]
+
+# Fuzzer
+[tasks.build]
+command = "cargo"
+args = ["+nightly", "fuzz", "build", "fuzz_target_1"]
+dependencies = [ "install_cargo_fuzz", "install_llvm_tools"]
+
+[tasks.test]
+linux_alias = "test_unix"
+mac_alias = "unsupported"
+windows_alias = "unsupported"
+
+[tasks.test_unix]
+script='''
+'''
```

Comment:
what we usually do is. 
run the fuzzer for 30 secs, grep the stdout, find the pattern "corpus: XXX".
if found then success, else no.